packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -232,17 +233,22 @@ export class MLSService {
     // We store the mapping between the subconversation and the parent conversation
     storeSubconversationGroupId(conversationId, subconversation.subconv_id, subconversation.group_id);
 
-    const keyLength = 256;
-    const groupIdBytes = Decoder.fromBase64(subconversation.group_id).asBytes;
-    const secretKey = await this.coreCryptoClient.exportSecretKey(groupIdBytes, keyLength);
+    const keyLength = 32;
+    const secretKey = await this.exportSecretKey(subconversation.group_id, keyLength);
 
     return {
       subconversation,
-      secretKey: new TextDecoder().decode(secretKey),
+      secretKey,
       keyLength,
     };
   }
 
+  public async exportSecretKey(groupId: string, keyLength: number): Promise<string> {
```

Comment:
maybe we could make it `private` first, WDYT?

Reply:
We will need it on webapp side when updating epoch info when handling `mls-message-add` backend event including `subconv: "conference"`. (Unless we will provide any other method that will have exporting this key included)

Reply:
I'm not yet sure how this update it going to happen, but there might be a async way that will warn the webapp that a new epoch and a new secret key was generated. 

let's keep it private for now and we could change it later on, as we go. 


Rule of thumb: Method should be private by default and it's trivial to make them public later on :)

Reply:
makes sense 👍